PassTime.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+include "classes.php";
+
+include "commonUI.php";
+
+
+$time = $_GET["restDays"];
```

Comment:
Make sure you run [intval()](http://php.net/manual/en/function.intval.php) on `$time`